gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
 modmenu-version = "3.0.0"
-cloth-config-fabric-version = "6.0.43"
-cloth-config-forge-version = "6.0.43"
```

Comment:
This also has to be changed in the forge's build.gradle. `fg.deobf` doesn't support versions catalog so it is duplicated there.